modules/microshift-greenboot-healthchecks-scripts.adoc

```diff
@@ -0,0 +1,11 @@
+// Module included in the following assemblies:
+//
+// * microshift_running applications/microshift-greenboot.adoc
+
+:_content-type: PROCEDURE
+[id="install-health-checks-greenboot_{context}"]
+== Installing health check script package
+
+Health check scripts are available in `/usr/lib/greenboot/check`, a read-only directory in RPM-OSTree systems. You must explicity install the following health check package `greenboot-default-health-checks`.
+
+//Q: procedure for this?
```

Comment:
No procedure. One just needs to bundle this RPM into the R4E ISO when building it.

Reply:
OK, so it sounds like something needs to be added to the main install docs for that. What about users who aren't using that method to install MicroShift? Is there a day 2 method to add the RPM?

Reply:
Yes, certainly. 
On `ostree` systems, the `microshift-greenboot` RPM can come via an update
On regular RHEL, running `dnf install microshift-greenboot` would have this effect.

Reply:
@fzdarsky Do we know where in Install we'd list this RPM as something to add? Is it a separate line item to do?

Reply:
Explained offline on how to update the docs.